src/main/kotlin/com/cognifide/gradle/aem/instance/InstanceState.kt

```diff
@@ -64,19 +66,27 @@ class InstanceState(private var syncOrigin: InstanceSync, val instance: Instance
                 return@check false
             }
 
-            val inactiveComponents = componentState.find(packagesActive, listOf()).filter { !it.active }
+            var result = true
```

Comment:
I prefer eager returns over mutating result variable. Why you changed it this way?